tools/commands.js

```diff
@@ -486,7 +486,21 @@ main.registerCommand({
     utils.validatePackageNameOrExit(
       packageName, {detailedColonExplanation: true});
 
-    var fsName = colonConverter.convert(packageName);
+    // When we create a package, avoid introducing a colon into the file system
+    // by naming the directory after the package name without the prefix.
+    var fsName = packageName;
+    if (packageName.indexOf(":") !== -1) {
+      var split = packageName.split(":");
+
+      if (split.length > 2) {
+        Console.error(packageName +
```

Comment:
Huh, interesting — who does that? They certainly aren't publishing them to troposphere.

In any case, it's a little weird to have an error like this one "Package name can't have more than one colon" when that actually is legal in other cases.

Honestly I would lean towards the stricter validation, but it sounds like you know about users who use this...  Otherwise, just take the last part?  I just don't like the idea of validation rules that are applied haphazardly in specific places.  (In fact, the reason we're having this conversation in the first place is because this exact rule is already applied haphazardly.)